charts/argocd/values.yaml

```diff
@@ -20,15 +20,17 @@ argo-cd:
                 Argo CD has not reported any applications data for the past 15 minutes which
                 means that it must be down or not functioning properly.  This needs to be
                 resolved for this cloud to continue to maintain state.
-          - alert: ArgoAppNotSynced
-            expr: |
-              argocd_app_info{sync_status!="Synced"} == 1
-            for: 12h
+          - alert: ArgoCDSync
+            expr: label_replace(argocd_app_info{sync_status!="Synced"}, "namespace", "$1", "dest_namespace", "(.*)") > 0
+            for: 60m
```

Comment:
I'm all for putting this to 60 mins, but I think it's better to discuss that this will not bother anyone else. Have you talked about this on our weekly, @omohammed95?

Reply:
We disscussed within the weekly we are all ok for one hour